libs/liblobby/lobby/lobby.lua

```diff
@@ -652,7 +652,7 @@ function Lobby:_OnBattleOpened(battleID, type, natType, founder, ip, port,
 	self.battles[battleID] = {
 		battleID = battleID, type = type, natType = natType, founder = founder, ip = ip, port = port,
 		maxPlayers = maxPlayers, passworded = passworded, rank = rank, mapHash = mapHash, spectatorCount = spectatorCount or 0,
-		engineName = engineName, engineVersion = engineVersion, mapName = mapName, title = title, gameName = gameName, users = {},
+		engineName = engineName, engineVersion = engineVersion, mapName = mapName, title = title, gameName = gameName, users = {founder},
```

Comment:
This could cause duplicate user bugs, or at least cause something to send warning about duplicate entries. Such things should be in the interface specific files. Add an "initial users" table which is sent to _OnBattleOpened and make the ZK interface send an empty table.